controllers/operator.go

```diff
@@ -410,6 +431,8 @@ func (o *operator) constructDeploy(partition int32) *appsv1.Deployment {
 	}
 	deploy.Annotations[PartitionAnnotation] = strconv.Itoa(int(partition))
 	deploy.Annotations[GenerationAnnotation] = o.observedGeneration()
+	deploy.ObjectMeta.Labels = map[string]string{ManagedByLabel: "true"}
```

Comment:
@lwolf  and @hagen1778  here, the deployment label get's set properly, but the pod label doesn't.

Deployment spec:
```bash
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  annotations:
    deployment.kubernetes.io/revision: "1"
    konsumerator.lwolf.org/generation: "177877537585038646"
    konsumerator.lwolf.org/partition: "54"
    konsumerator.lwolf.org/scaling-status: RUNNING
    konsumerator.lwolf.org/scaling-status-change: "2020-03-09T22:51:47-07:00"
  creationTimestamp: "2020-03-10T05:51:47Z"
  generation: 1
  labels:
    konsumerator.lwolf.org/managed-by: "true"
...
```
Have you observed this behavior before? I've not been able figure this one out and I'm stuck. For what's it worth I have also tried the following alternatives:

```go
	deploy.Spec.Template.SetLabels(map[string]string{ManagedByLabel: "true"})
	deploy.Spec.Template.Labels = map[string]string{ManagedByLabel: "true"}
```

I noticed that while testing out if the labels are getting set and if I'm able to collect pods having matching labels -- they don't. I might be missing something really trivial.

Reply:
This seems to be the case still. But like @lwolf  suggested, I was able to set the labels on the consumer (used the files under `hack/ci`) directly and was able to use a bunch of debug statements to verify that fetching the pods by label works :) ! Also deployed the `faker` to trigger reconciliation.